test/gtest/uct/test_md.cc

```diff
@@ -276,7 +280,8 @@ void test_md::dereg_cb(uct_completion_t *comp)
 bool test_md::is_gpu_ipc() const
 {
     return (GetParam().md_name == "cuda_ipc") ||
-	   (GetParam().md_name == "rocm_ipc");
+           (GetParam().md_name == "rocm_ipc") ||
+           (GetParam().md_name == "gdr_copy");
```

Comment:
why is it needed? it's not really a gpu_ipc transport

Reply:
can you update the test instead. E.g. like this:
query md attrs with `uct_md_query_v2`
skip the test if `md_attr->reg_mem_types` does not contain host memory

Reply:
Right, as @brminich the right way to to check if the MD can register host memory .
Maybe after this, we could remove is_gpu_ipc condition as well to cleanup the code

Reply:
Ok, agree, will be fixed